lib/aur-build

```diff
@@ -373,7 +407,7 @@ while IFS= read -ru "$fd" path; do
                 (( dry_run )) && printf '%s:%s:%s\n' build "$pkgbase" "file://$pkgpath"
             fi
         # pkgbase may differ from pkgname; prefix to package path with --full
-        done < <(PKGDEST="$db_root" aur build--pkglist --full "${pkglist_args[@]}")
+        done < <(as_user PKGDEST="$db_root" aur build--pkglist --full "${pkglist_args[@]}")
```

Comment:
As `setpriv` runs the command without a shell are you sure `as_user PKGDEST=...` works as intended, should it perhaps not be: `PKGDEST=... as_user ...`?  The `as_user` function uses `env` but the `"$@"` comes after the terminating `--`.

Reply:
These are the possible combinations:
```
$ foo=1 env printenv foo
1

$ env foo=1 printenv foo
1

$ env foo=1 -- printenv foo
env: ‘--’: No such file or directory

$ env -- foo=1 printenv foo
1

$ env -C ~ -- foo=1 printenv foo
1

$ env -- -C ~ foo=1 printenv foo
env: ‘-C’: No such file or directory
```
so it's perhaps unexpected, but it works